app/styles/_base.scss

```diff
@@ -150,6 +150,42 @@ input {
   font-weight: inherit;
 }
 
+input[type='radio'] {
+    @include font();
+    
```

Comment:
You can include the comment from that other `appearance` usage in the other CSS file. (where it said it that `autoprefixer` does not support `appearance`)